### PR TITLE
Update lab-06-2-softmax_zoo_classifier.py

### PR DESCRIPTION
**Pull request by BlueMelon715**
lab-06-2-softmax_zoo_classifier.py에서 tf.nn.softmax과 tf.nn.softmax_cross_entropy_with_logits가 같이 사용되어 개념이 혼동되어 Tensorflow 공식페이지를 찾아보았습니다. tf.nn.softmax_cross_entropy_with_logits가 내부적으로 softmax를 적용시키므로 tf.nn.softmax를 같이 적용시키지 않는다는 것을 확인했습니다.

참고1 : https://www.tensorflow.org/get_started/mnist/beginners
참고2 : https://www.tensorflow.org/get_started/mnist/pros
참고3 : https://www.facebook.com/groups/TensorFlowKR/permalink/399110030430061/?comment_id=399110370430027&reply_comment_id=399110850429979

### CODE REVIEW

File: lab-06-2-softmax_zoo_classifier.py [hunk 1]
```diff
@@ -27,10 +27,9 @@
 hypothesis = tf.nn.softmax(logits)
 
 # Cross entropy cost/loss
-cost_i = tf.nn.softmax_cross_entropy_with_logits(logits=logits,
```
Comment (hunkim) [inline lab-06-2-softmax_zoo_classifier.py]: That's why we passed logits (not hypothesis). Do you think this is not correct?

Comment (BlueMelon715) [inline lab-06-2-softmax_zoo_classifier.py]: I'm sorry. I overlooked that point. It is correct. Thank you very much.

**Comment by hunkim** [comment]
I really appreciate your PR. Please feel free to give us other PRs.

**Comment by BlueMelon715** [comment]
Yes, I will. Thank you!

2017. 3. 13. 오후 10:13에 "Sung Kim" <notifications@github.com>님이 작성:
